stl/inc/limits

```diff
@@ -1168,6 +1180,24 @@ _NODISCARD _CONSTEXPR20 int _Popcount(const _Ty _Val) noexcept {
     return _Popcount_fallback(_Val);
 }
 
+template <typename _Ty, typename _Fn>
+auto _Select_popcount_impl(_Fn _Callback) {

```

Comment:
`decltype(auto)` would be a bit more generic here, allowing `_Callback` to return reference types without decaying or even `void`. The existing caller admittedly doesn't need that genericity.